Branch-SDK/Branch-SDK/Requests/BranchShortUrlSyncRequest.m

```diff
@@ -84,7 +84,9 @@ - (NSString *)createLongUrlForUserUrl:(NSString *)userUrl {
 }
 
 + (NSString *)createLinkFromBranchKey:(NSString *)branchKey tags:(NSArray *)tags alias:(NSString *)alias type:(BranchLinkType)type matchDuration:(NSInteger)duration channel:(NSString *)channel feature:(NSString *)feature stage:(NSString *)stage params:(NSDictionary *)params {
-    NSMutableString *baseUrl = [[NSMutableString alloc] initWithFormat:@"%@/a/%@?", BNC_LINK_URL, branchKey];
+    BNCPreferenceHelper *preferenceHelper = [BNCPreferenceHelper preferenceHelper];
+    NSMutableString *baseUrl = [preferenceHelper.userUrl mutableCopy];
```

Comment:
Because for non-bnc.lt accounts, this will create links in the form of bnc.lt/a/key_livexx/blah whereas the userUrl will have the correct domain. My change will fail if a session was never initialized and currently will return an empty link